content/docs/command-reference/add.md

```diff
@@ -153,10 +153,12 @@ not.
   > Additionally, this typically requires an external cache setup (see link
   > above).
 
-- `-o <path>`, `--out <path>` - destination `path` to make a local target copy,
-  or to [transfer](#example-transfer-to-cache) an external target into the cache
-  (and link to workspace). Note that this can be combined with `--to-remote` to
-  avoid storing the data locally, while still adding it to the project.
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to link
+  (or copy) a data target, which will now be tracked by DVC. Note that combining
+  this with an
+  [external cache transfer](#example-transfer-to-an-external-cache), or with the
```

Comment:
external cache probably is not the best term here - remote storage is better

Reply:
should it link to the ` Transfer to remote storage`?

Reply:
This whole note was already removed per https://github.com/iterative/dvc.org/pull/2302#pullrequestreview-612504838 above.